src/controllers/users.go

```diff
@@ -19,9 +20,16 @@ func NewUsersController(db *gorm.DB) *UsersController {
 
 // Create new user
 func (c *UsersController) Create(email string, password string) (*models.User, error) {
+
+	hashedPassword, err := utils.HashPassword(password)
+
+	if err != nil {
```

Comment:
this call should not fail (under any normal operation)
Can you do a `lo.Must` wrapper simply instead ?